mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/task/SolveMessageInconsistenciesService.java

```diff
@@ -162,6 +165,23 @@ private void notifySuccess(Context context) {
         }
     }
 
+    public static class RunningOptions {
+
+        public static final RunningOptions DEFAULT = new RunningOptions(10);
```

Comment:
Here we are only processing metadata and never loading message content. I would advocate for a default value of at least 100.